CHANGES.md

```diff
@@ -1,5 +1,66 @@
 # Changelog
 
+# 1.0.0
+
+## Major Changes
+
+* A docs site overhaul! Along with tons of additional content, the existing pages have been significantly edited and reorganized to improve readability.
+* All Dagster [examples](https://github.com/dagster-io/dagster/tree/master/examples)[](https://github.com/dagster-io/dagster/tree/master/examples)are revamped with a consistent project layout, descriptive names, and more helpful README files.
+* A new `dagster project `CLI contains commands for bootstrapping new Dagster projects and repositories
+    * `dagster project scaffold` creates a folder structure with a single Dagster repository and other files such as workspace.yaml. This CLI enables you to quickly start building a new Dagster project with everything set up.
+    * `dagster project from-example` downloads one of the Dagster examples. This CLI helps you to quickly bootstrap your project with an officially maintained example. You can find the available examples via `dagster project list-examples`.
+    * Check out [Create a New Project](https://docs.dagster.io/getting-started/create-new-project) for more details.
+* A `default_executor_def` argument has been added to the `@repository` decorator. If specified, this will be used for any jobs (asset or op) which do not explicitly set an `executor_def`.
+* A `default_logger_defs` argument has been added to the `@repository` decorator, which works in the same way as `default_executor_def`.
+* A new `execute_job` function presents a new Python API for kicking off runs of your jobs.
```

Comment:
"new" twice is a little weird? not a big deal though

Reply:
good catch